week1/main.py

```
@@ -0,0 +1,48 @@
+
+# Initialize DataProcessor
+data_processor = DataProcessor('data/data.csv', config)
+logger.info("DataProcessor initialized.")
+
+# Preprocess the data
+data_processor.preprocess_data()
+logger.info("Data preprocessed.")
+
+# Split the data
+X_train, X_test, y_train, y_test = data_processor.split_data()
+logger.info(f"Data split into training and test sets.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)